removed Visual Studio (#5062)
umbraco/UmbracoDocs: umbraco-cloud/set-up/working-locally.md

> * Git needs to be installed on your computer

Reviewer: ```suggestion
* Git needs to be installed on your computer in order to clone down the project and push your changes up to Cloud.
```

>   * Example: [Fork](https://git-fork.com/), [SourceTree](https://www.sourcetreeapp.com/), or [GitKraken](https://www.gitkraken.com/)

Reviewer: ```suggestion
  * We recommend using one of the following git-clients if you are new to git:: [Fork](https://git-fork.com/), [SourceTree](https://www.sourcetreeapp.com/), or [GitKraken](https://www.gitkraken.com/)
```

> 

Reviewer: ```suggestion
* Visual Studio - for running the project on your local machine.
```
